src/renderer/lib/state.js

```diff
@@ -107,7 +107,8 @@ function setupSavedState (cb) {
       downloadPath: config.DEFAULT_DOWNLOAD_PATH,
       isFileHandler: false,
       openExternalPlayer: false,
-      externalPlayerPath: null
+      externalPlayerPath: null,
+      startup: true
```

Comment:
Why not? There's a pref to disable

Looks like uTorrent not only starts automatically, but it re-enables that option after every auto-update (!) https://forum.utorrent.com/topic/92281-disable-utorrent-program-auto-start-as-default-setting/

Obv we don't want to do that---if someone turns off the pref, we'll respect it permanently---but I think auto start after they first install WebTorrent is fine.